extensions/todoist/src/components/MenubarTask.tsx

```diff
@@ -0,0 +1,120 @@
+import { Task } from "@doist/todoist-api-typescript";
+import { Color, confirmAlert, Icon, MenuBarExtra, open, showHUD } from "@raycast/api";
+import { MutatePromise } from "@raycast/utils";
+import { useEffect } from "react";
+import { todoist } from "../api";
+import { priorities } from "../constants";
+import { checkTodoistApp, isTodoistInstalled } from "../helpers/isTodoistInstalled";
+
+interface MenubarTaskProps {
+  task: Task;
+  mutateTasks: MutatePromise<Task[] | undefined>;
+}
+
+const MenubarTask = ({ task, mutateTasks }: MenubarTaskProps) => {
+  useEffect(() => {
+    checkTodoistApp();
+  }, []);
```

Comment:
Isn't it possible to wrap the command into the `View` component to avoid performing the check?

Reply:
Not sure how it is persisted. For example if one installs the extension and then uses none of the other commands except for the menu bar one, this will definitely not work.

Reply:
> For example if one installs the extension and then uses none of the other commands except for the menu bar one, this will definitely not work.

Why would it not work?

Reply:
Oh sorry, I thought you were referring to the value if `isTodoistInstalled` being persisted when it is checked for by any of the other commands. Wrapping `MenubarTask` in `View` works, I have added that in https://github.com/raycast/extensions/pull/3597/commits/6f104b73a79766d17e5688d651fed8a70c449eb3